lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback.inc

```diff
@@ -333,7 +333,7 @@ class ReportbackEntityController extends EntityAPIController {
     }
 
     parent::save($entity, $transaction);
-
+    
```

Comment:
as I noted in the PR description, this used to be there but I accidentally removed it before. I guess I'll remove it again haha